cs.html

```diff
@@ -554,6 +695,102 @@ <h2>Dictionary BidirectionalStreamEventInit Members</h2>
         </section>
       </div>
     </section>
+    <section>
+      <h3><dfn>DatagramsReceivedEvent</dfn></h3>
+      <p>The <code><a>datagramsreceived</a></code> event uses the
+      <code><a>DatagramsReceivedEvent</a></code> interface.</p>
+      <div>
+        <pre class="idl">
+        [ Constructor (DOMString type, DatagramsReceivedEventInit eventInitDict), Exposed=Window]
+interface DatagramsReceivedEvent : Event {
+    readonly        attribute sequence<Uint8Array?> datagrams;
```

Comment:
There's a warning on the page:
"No <dfn> for attribute datagrams in DatagramsReceivedEvent."

https://rawgit.com/w3c/webrtc-quic/datagram/cs.html#dom-datagramsreceivedevent

Reply:
Hopefully fixed now